embeddings/evaluator/metrics_evaluator.py

```diff
@@ -26,6 +26,6 @@ def metrics(
 
     @abc.abstractmethod
     def evaluate(
-        self, data: Union[Dict[str, nptyping.NDArray[Any]], Predictions]
+        self, data: Union[Dict[str, nptyping.NDArray[Any]], Predictions, Dict[str, Any]]
```

Comment:
What is the reason of adding `Dict[str, Any]]` as we should omit `Any` as much as possible?